pkgs/development/python-modules/jsonrpclib-pelix/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "jsonrpclib-pelix";
+  version = "0.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1qs95vxplxwspbrqy8bvc195s58iy43qkf75yrjfql2sim8b25sl";
+  };
+
+  meta = with lib; {
+    description = "JSON RPC client library - Pelix compatible fork";
+    homepage = https://pypi.python.org/pypi/jsonrpclib-pelix/;
+    license = lib.licenses.asl20;
+    maintainers = [ lib.maintainers.joachifm ];
```

Comment:
Is @joachifm maintaining this expression?

Reply:
They haven't replied yet. What's the procedure for finding a maintainer for necessary new dependencies?

Reply:
You can make yourself the maintainer or leave it empty (preference on the former).

Reply:
I've removed the maintainer line for now, I'm not sure I can provide the level of attention needed atm.

Reply:
I only care to the extent that it affects electrum, so wouldn't be maintaining it per se. It used to be internal to the electrum expression for that reason.

Reply:
I've added myself as maintainer afterall.